src/library/Parser/MusicBrainzChapterParser.php

```diff
@@ -54,6 +54,11 @@ public function loadRecordings($retries=5, $pause=100000, $callback = 'file_get_
                 ];
 
                 $context = stream_context_create($options);
+                $headers = get_headers($urlToGet, false, $context);
+                if (substr($headers[0], 9, 3) === "404") {
```

Comment:
This is kind of prone to error... but I don't have a better solution, so I'll allow it :-)

Reply:
Yep. But I don't think there's a better solution, and PHP should already check that it's a proper HTTP status line